### ISSUE TITLE
Move registering for remote events out of LibraryViewController

### ISSUE THREAD

**Issue by pichfl** [GianniCarlo/Audiobook-Player#68]
We should move this to a separate service class which in turn can be called from AppDelegate, which seems like a more appropriate place to me. But I can be wrong, please advise. 

Goal should be to make the purpose of the LibraryViewController more distinct to make it easier for the new features to be implemented.

**Comment by GianniCarlo** [comment]
Agreed, AppDelegate sounds good for now. Besides moving the remote events, we could also move these events:
- `AVAudioSessionInterruption`
- `AVAudioSessionRouteChange`
